ipn/ipntest/server.go

```diff
@@ -0,0 +1,324 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+// Package ipntest provides utilities for black-box testing of LocalAPI ([ipnserver]).
```

Comment:
This adds another use of "IPN" which means nothing. I'd kinda rather phase it out, if anything.

localapitest? or "lapitest" if we want shorter?

Because technically the "ipnserver" package isn't really about LocalAPI ... originally it was about HTTP-ifying the "IPN API" when the IPN bus/API was all a custom JSON streaming protocol.